RetailCoder.VBE/Root/RubberduckModule.cs

```diff
@@ -306,6 +306,11 @@ private void BindQuickFixTypes(IEnumerable<Assembly> assemblies)
             foreach (var quickFix in quickFixes)
             {
                 Bind<IQuickFix>().To(quickFix).InSingletonScope();
+                if (quickFix.Name.Equals("AssignedByValParameterMakeLocalCopyQuickFix"))
```

Comment:
That's probably because it follows the rules of interfaces instead of factories.